packages/swarm-styles/lib/spacing.css

```diff
@@ -1,3 +1,17 @@
 :root {
-    --space: 16px;
-}
\ No newline at end of file
+    --space: var(--size-space-default);
+}
+
+/* Medium breakpoint overrides */
+@media only screen and (min-width: 640px) {
+  :root {
+      --responsive-space: var(--size-space-largeDefault);
```

Comment:
This would have all space sizes assigned so that they are applied within the media query.

```
--space: --size-space-largeDefault;
--space-double: --size-space-largeDouble;
...etc
```